build/psgallery-release.yml

```diff
@@ -150,7 +150,14 @@ stages:
             displayName: 'Install NuGet'
           - powershell: |
               Get-ChildItem -Path $env:ARTIFACT_DIR -Filter *.nupkg -Recurse |
-                % { & "nuget" push $_.FullName -Source $(Source) -ApiKey $(NuGet.ApiKey) -SkipDuplicate }
+                % { & "nuget" push $_.FullName -Source $(Source) -ApiKey $(NuGet.ApiKey) -SkipDuplicate | Out-File log.txt -Append }
+              
+              $log = Get-Content log.txt
+              cat $log
+              if ($log -match "BadRequest") {
+                echo "##vso[task.logissue type=error]Not all PowerShell modules were pushed correctly to the PowerShell Gallery"
+                exit 1
+              }
```

Comment:
We log during release, and verify at the end.
Would want to check this with a pre-release with both a invalid and valid release version, maybe. Not sure how otherwise we could test this 🤔 .

Reply:
Difficulty is in how to test indeed